src/styles/main.css

```diff
@@ -66,16 +74,54 @@ li > a:hover {
   color: hsl(0, 100%, 30%);
 }
 
+.cards-display-section {
+  margin-bottom: 8rem;
+}
+
+.movie-item {
+  background-color: hsl(0, 100%, 30%);
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding: 10px;
+}
+
+.movie-title {
+  margin: 5px 0;
+  color: hsl(0, 100%, 30%);
+}
+
+.btn {
+  color: hsl(0, 100%, 30%);
+  border: none;
+  background-color: #000;
+  padding: 10px;
+  width: 100%;
+}
+
 footer {
   display: flex;
-  justify-content: flex-end;
+  justify-content: center;
   align-items: center;
   padding-right: 2rem;
   padding-left: 2rem;
-  height: 50px;
+  height: 100px;
   background-color: #000;
   color: #fff;
   position: fixed;
   bottom: 0;
   width: 100%;
 }
+
+footer p {
+  padding: 5px;
+  font-weight: 700;
+  font-size: large;
+}
+
+@media screen and (max-width: 768px) {
+  #logo {
+    margin-right: 1.5rem;
+    width: 50%;
+    height: 50%;
+    margin-top: 1rem;
+  }
+}
```

Comment:
There is a problem in the logo with the screen size, you can see it in the picture below, the logo image size is not responsive to the size of the screen,
<img width="527" alt="image" src="https://github.com/Zohra-Neda/JavaScript-Capstone-Project/assets/118947090/b89ccd96-b1ec-46c5-9c29-379d327a56be">

Reply:
Hi @Nahid1911 ,
Thanks for the constructive feedback, when I open it on my browser and then click on mobile icon it fixed automatically on different screen sizes 😊 ✔